.prettierignore

```diff
@@ -0,0 +1,8 @@
+vendor
+**/static
+*.html
+*.css
+*.md
+*.yaml
+*.yml
+*.scss
```

Comment:
Is there a reason we _shouldn't_ use prettier on these file types as well?

Reply:
i added markdown and yml

html is done by djlint
css/scss is done by stylelint.

Otherwise we will have to deal with formatter wars if we have multiple